CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Terminal escape bindings with combined modifiers for Delete and Insert
 - /Applications symlink into OS X DMG for easier installation
 - Colored emojis on Linux/BSD
+- `Minimize` action
```

Comment:
We should probably go into a bit more detail here, I doubt anyone would understand what this means by just reading the change log entry.

```suggestion
- `Minimize` key binding action, bound to `cmd + m` on macOS
```

Reply:
I was just using similar description to the one from `v0.3.3` added section about `ToggleFullscreen` action.

I'll update once Windows situation becomes more clear.